bin/repl.js

```diff
@@ -31,15 +46,17 @@ scxml.pathToModel(pathToScxml,function(err,model){
     //Use the statechart object model to instantiate an instance of the statechart interpreter. Optionally, we can pass to the construct an object to be used as the context object (the 'this' object) in script evaluation. Lots of other parameters are available.
     var interpreter = new scxml.scion.Statechart(model, interpOpts);
 
+
+    interpreter.registerListener(listeners);
+
+
     interpreter.start();
 
     console.log(interpreter.getConfiguration());
 
-    var parseRE = /\((.*)\n\)/;
-
     function processEvent(cmd,dontKnow,alsoDontKnow,callback){
-        var e = cmd.match(parseRE)[1];
-        interpreter.gen({name : e});
+        cmd = cmd.trim();
+        interpreter.gen({name : cmd});
         var conf = interpreter.getConfiguration();
```

Comment:
It seemed cumbersome to require each fsm input to be surrounded by parens. If it's important I can restore it and add an explanatory comment that you provide.